renderer/components/Editor.tsx

```diff
@@ -628,6 +694,14 @@ export class Editor extends React.Component<Props, State> {
                   </DropdownButton>
                 </OverlayTrigger>
               </InputGroup>
+              <TooltipButton
+                    id="toggleKeyboardControl"
+                    text="toggleKeyboardControl"
```

Comment:
Let's follow the convention we have from other texts and do `Toggle Keyboard Control Mode`